bitcoin/src/bip32.rs

```diff
@@ -473,6 +475,7 @@ impl fmt::Display for Error {
                 write!(f, "encoded extended key data has wrong length {}", len),
             Error::Base58(ref e) => write_err!(f, "base58 encoding error"; e),
             Error::Hex(ref e) => write_err!(f, "Hexadecimal decoding error"; e),
+            Error::InvalidPublicKeyHexLength(got) => write!(f, "PublicKey hex should be 66 or 130 digits long, got: {}", got),
```

Comment:
I think we should use user friendly descriptions rather than type names in error messages but I guess there are other instances, so we may want to do complete audit and cleanup.

Reply:
Since we are on the topic, error messages should not be capitialised, right? I always forget and we have so many of both forms.

Reply:
Noted however not to use types in error messages.

Reply:
> error messages should not be capitialised, right?

I think so.